yandexcloud/_channels.py

```diff
@@ -1,14 +1,18 @@
 import grpc
-import pkg_resources
 
 from yandex.cloud.endpoint.api_endpoint_service_pb2 import ListApiEndpointsRequest
 from yandex.cloud.endpoint.api_endpoint_service_pb2_grpc import ApiEndpointServiceStub
 from yandexcloud import _auth_plugin
 from yandexcloud._auth_fabric import YC_API_ENDPOINT, get_auth_token_requester
 
 try:
-    VERSION = pkg_resources.get_distribution("yandexcloud").version
-except pkg_resources.DistributionNotFound:
+    from importlib.metadata import PackageNotFoundError, version
+except ImportError:
+    from importlib_metadata import PackageNotFoundError, version
```

Comment:
@90victor09, hi! Thanks for your PR

What's the purpose of executing the same line in `except` section? Wouldn't it throw the same `ImportError`?